pkg/command/utils/util.go

```diff
@@ -72,3 +72,18 @@ func GenerateJSONFile(jsonData []byte, targetJSON string) error {
 	}
 	return nil
 }
+
+func CheckOutputLocation(outputLocation string) (string, error) {
+	if outputLocation[len(outputLocation)-1:] == "/" {
+		outputLocation = outputLocation[0 : len(outputLocation)-1]
+	}
+	dirInfo, err := os.Stat(outputLocation)
+	if err != nil {
+		return outputLocation, fmt.Errorf("output location (%s) is not existed: %s\n", outputLocation, err)
```

Comment:
Only when `os.IsNotExist(err)` is true, that means the directory does not exist.
```
if os.IsNotExist(err) {
        // file does not exist
    } else {
        // other error
    }
```

Reply:
Changed the code as you suggested.